gtk4/src/subclass/color_chooser.rs

```diff
@@ -13,12 +13,80 @@ pub trait ColorChooserImpl: ObjectImpl {
         orientation: Orientation,
         colors_per_line: i32,
         colors: &[RGBA],
-    );
-    fn color_activated(&self, color_chooser: &Self::Type, rgba: RGBA);
+    ) {
+        self.parent_add_palette(color_chooser, orientation, colors_per_line, colors);
+    }
+
+    fn color_activated(&self, color_chooser: &Self::Type, rgba: RGBA) {
+        self.parent_color_activated(color_chooser, rgba);
+    }
+
     fn get_rgba(&self, color_chooser: &Self::Type) -> RGBA;
     fn set_rgba(&self, color_chooser: &Self::Type, rgba: RGBA);
```

Comment:
And functions that don't have default impls still should get a function to chain up on the `Ext` trait. The parent class might implement the interface already and the subclass might want to call that.

Reply:
Ah, right. I somehow missed that :-)